guide.md

```diff
@@ -45,45 +34,139 @@ We designed this guide to help anyone navigate those complexities—regardless o
 
 ---
 
-### [Getting started]({{ '/guide/getting-started/introduction/' | relative_url }})
+<div class="columns-desktop -two">
+<div class="column" markdown="1">
 
-Provides top-level overviews of common visual language, technology, software, and hardware.
+<h2 class="h3" markdown="1">[Getting started]({{ '/guide/getting-started/introduction/' | relative_url }})</h2>
 
----
+New to bitcoin? This section provides top-level overviews of [open design]({{ '/guide/designing-products/open-design/' | relative_url }}), [visual language]({{ '/guide/getting-started/visual-language/' | relative_url }}), [design principles]({{ '/guide/designing-products/principles/' | relative_url }}), a [technology primer]({{ '/guide/getting-started/technology-primer/' | relative_url }}) and more.
 
-### [Designing bitcoin products]({{ '/guide/designing-products/introduction/' | relative_url }})
+</div>
+<div class="column" markdown="1">
 
-A close look at the design process, from the concept of open design to use cases and frameworks like the usage life cycle.
+<h2 class="h3" markdown="1">[Designing bitcoin products]({{ '/guide/designing-products/introduction/' | relative_url }})</h2>
+
+A closer look at the design process and framworks, from [personal finance use cases]({{ '/guide/designing-products/personal-finance/' | relative_url }}) to [units & symbols]({{ '/guide/payments/units-and-symbols/' | relative_url }}), the [usage life cycle]({{ '/guide/designing-products/usage-life-cycle/' | relative_url }}), [user research]({{ '/guide/designing-products/user-research/' | relative_url }}), and more.
+
+</div>
+</div>
 
 ---
 
-### [Onboarding]({{ '/guide/onboarding/introduction/' | relative_url }})
+<h4 class="h3" markdown="1">Reference designs</h4>
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending.png"
+   retina = "/assets/images/guide/daily-spending@2x.png"
+   imageMobile = "/assets/images/guide/daily-spending-mobile.png"
+   retinaMobile = "/assets/images/guide/daily-spending-mobile@2x.png"
+   alt-text = "Strong handshake"
+   width = 1600
+   height = 840
+%}
+
+<h2 class="h3" markdown="1">[Daily spending wallet]({{ '/guide/daily-spending-wallet/' | relative_url }})</h2>
+
+An in-depth exploration of a mobile wallet for a Lightning-first, on-the-go use case. Covers primary user flows like [first use]({{ '/guide/onboarding/first-use/' | relative_url }}), [sending]({{ '/guide/payments/send/' | relative_url }}) and [requesting]({{ '/guide/payments/request/' | relative_url }}), features like [backup]({{ '/guide/onboarding/backing-up-a-wallet/' | relative_url }}) and [contacts]({{ '/guide/payments/contacts/' | relative_url }}), and more.
```

Comment:
```suggestion
An in-depth exploration of a bitcoin-only, non-custodial, mobile lightning-first, wallet for on-the-go payments. Covers primary user flows like [first use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}), [sending]({{ '/guide/daily-spending-wallet/sending/' | relative_url }}) and [requesting]({{ '/guide/daily-spending-wallet/requesting/' | relative_url }}), features like [backup]({{ '/guide/daily-spending-wallet/backup-and-recovery/' | relative_url }}) and [contacts]({{ '/guide/daily-spending-wallet/contacts/' | relative_url }}), and more.
```
Just wanted to make the use-case a bit more explicit

Reply:
I think it's too much. Very hard to read this way.